helix-core/src/main/java/org/apache/helix/controller/stages/CurrentStateOutput.java

```diff
@@ -238,6 +256,16 @@ public Message getPendingMessage(String resourceName, Partition partition, Strin
     return getStateMessage(resourceName, partition, instanceName, _pendingMessageMap);
   }
 
+  public Set<Message> getStaleMessages(String instanceName) {
```

Comment:
nit, maybe call it getStaleMessagesByInstance would help to differentiate it from getStaleMessageMap